src/modules/EditProfileModule/index.tsx

```
@@ -0,0 +1,106 @@
+      const response = await axios.put(
+        `/users/${formData.id}`, 
+        updatedProfile,
+        {
+          headers: {
+            'Authorization': `Bearer ${token}`,
+            'Content-Type': 'application/json'
+          }
+        }
+      );
+      
+      if (response.data && response.data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)